core/src/main/java/io/questdb/griffin/engine/AbstractRedBlackTree.java

```diff
@@ -62,13 +64,14 @@ public void close() {
     }
 
```

Comment:
Fixes tree size calculation bug caused by small amounts of memory (smaller than node size) being left at end of each page .